java/build.gradle

```
@@ -16,9 +16,13 @@ repositories {
 
 def grpcVersion = '1.43.2'
 def protobufVersion = '3.19.2'
+def slf4jVersion = '1.7.36'
+def log4jVersion = '2.19.0'
 
 dependencies {
-    compile group: 'org.slf4j', name: 'slf4j-log4j12', version: '1.7.30'
+    compile group: 'org.slf4j', name: 'slf4j-api', version: "${slf4jVersion}"
+    compile group: 'org.apache.logging.log4j', name: 'log4j-core', version: "${log4jVersion}"
+    compile group: 'org.apache.logging.log4j', name: 'log4j-slf4j-impl', version: "${log4jVersion}"
```

I think we should use `log4j-slf4j2-impl` for SLF4j 2.0.
```suggestion
    compile group: 'org.apache.logging.log4j', name: 'log4j-slf4j2-impl', version: "${log4jVersion}"
```

---

Ah, sorry. I misunderstood this PR upgrades slf4j to 2.0. Please ignore it.